data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg

```diff
@@ -30,291 +31,707 @@ _"Nym is Kaleh’s childhood friend, a young, rebellious lady with quick wits an
+        require_amla = swordsmanship_training
+        {COMMON_AMLA_EFFECTS}
         [effect]
-            apply_to=attack
-            name=bow
-            increase_damage=2
-            increase_attacks=1
+            apply_to = attack
+            name = sword
+            increase_damage = 2
+        [/effect]
+        [effect]
+            apply_to = hitpoints
+            increase_total = 1
+            heal_full = yes
         [/effect]
     [/advancement]
     [advancement]
-        id=hunter_1
-        description= _ "Hunter"
-        image=attacks/bolas.png
+        id = swordsman
+        description = _"Swordsman"
+        image = icons/crossed_sword_and_hammer.png
+        require_amla = strong_strikes
+        {COMMON_AMLA_EFFECTS}
         [effect]
-            [filter]
-                type=Quenoth Youth
-            [/filter]
-            apply_to=type
-            name=Quenoth Youth 2
+            apply_to = attack
+            name = sword
+            increase_damage = 1
         [/effect]
         [effect]
-            [filter]
-                type=Quenoth Youth 2
-            [/filter]
-            apply_to=type
-            name=Quenoth Youth 3
+            apply_to = attack
+            name = sword
+            increase_attacks = 1
         [/effect]
-        {HEAL_EFFECTS}
-        {EXTRA_EFFECTS}
         [effect]
-            apply_to=new_attack
-            name=bolas
-            description= _"bolas"
-            type=impact
-            range=ranged
-            damage=10
-            number=2
-            icon=attacks/bolas.png
+            apply_to = hitpoints
+            increase_total = 2
+            heal_full = yes
+        [/effect]
+    [/advancement]
+    [advancement]
+        id = taunt
+        description = _"Taunt"
+        image = attacks/fangs-horse.png
+        require_amla = swordsmanship_training
+        {COMMON_AMLA_EFFECTS}
+        [effect]
+            apply_to = max_attacks
+            increase = 1
+        [/effect]
+        [effect]
+            apply_to = new_attack
+            name = taunting_strike
+            #textdomain wesnoth-units
+            description= _"taunting strike"
+            type = blade
+            range = melee
+            damage = 7
+            number = 1
+            icon = attacks/sword-elven.png
+            attacks_used = 1
             [specials]
-                {WEAPON_SPECIAL_SLOW}
+                {WEAPON_SPECIAL_USES_ATTACKS 1}
+                {WEAPON_SPECIAL_TAUNT}
             [/specials]
+            parry = 10
         [/effect]
         [effect]
-            apply_to=movement
-            increase=1
+            apply_to = hitpoints
+            increase_total = 4
+            heal_full = yes
         [/effect]
     [/advancement]
     [advancement]
-        id=leader_1
-        description= _ "Leader"
-        image=attacks/leader.png
+        id = run_by_taunt
+        description = _"Run By (Taunting strike)"
```

Comment:
I've received word from a translator having trouble understanding this description; please add a `# po:` comment to explain how it works a bit better